lang/python/python3-importlib-metadata/patches/001-fix-scm-version.patch

```diff
@@ -0,0 +1,10 @@
+Index: importlib_metadata-0.17/setup.py
+===================================================================
+--- importlib_metadata-0.17.orig/setup.py
++++ importlib_metadata-0.17/setup.py
```

Comment:
Why it doesn't look only for `setup.py` but it's looking for `importlib_metadata-0.17/setup.py`?

Reply:
Much better, thank you! :)